content/en/blog/2023/http-conventions-declared-stable.md

```diff
@@ -0,0 +1,271 @@
+---
+title: HTTP semantic conventions declared stable
+linkTitle: HTTP semconv are stable
+date: 2023-11-03
+author: '[Trask Stalnaker](https://github.com/trask) (Microsoft)'
+cSpell:ignore: Liudmila Molkova
+---
+
+Early this year we embarked on a three month project to stabilize HTTP semantic
+conventions. We are proud to announce (nine months later), that the HTTP
+semantic conventions are now the first OpenTelemetry semantic convention to be
+declared stable! The stable version is a significant evolution over the previous
+unstable version:
+
+- Improvements brought over from the
+  [alignment with Elastic Common Schema](/blog/2023/ecs-otel-semconv-convergence/)
+  - The `url.*` namespace, which can be reused in the future by non-HTTP
+    semantic conventions
+  - The `client.*` and `server.*` namespaces, which are easier to reason about
+    and work better on logs (where there is no span kind)
```

Comment:
What exactly does that mean?

Reply:
I've expanded this bullet, lmk if it helps